### PR TITLE
[prometheus] correct NOTES.txt for pod labels selection

### PR DESCRIPTION
**Pull request by tdihp**
#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by zanhsieh — COMMENTED** [review]
@tdihp 
Thank you for the contribution. Can you bump the version in Chart.yaml so CI can merge this PR once approved?

**Comment by tdihp** [comment]
Thanks @zanhsieh , I've updated the chart version, admittedly during this I spotted more issues, I tracked those in #3490.

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**
